CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.0 (2018-11-28)
+
+- Added `.promise()` method to wrap Ora Library functionality. [#6](https://github.com/blackbaud/skyux-logger/pull/6)
```

Comment:
It's probably fine as is, but should `.promise()` be `logger.promise()`? ... I also think "library" should be lowercase, so I'll go ahead and change that.

Reply:
In this case, `logger` would be whatever variable they import our library as.  For example, a consumer could do the following:

```
const theBlackbaudLogger = require('@blackbaud/skyux-logger');
theBlackbaudLogger.promise().
...
```